roles/nova-common/templates/etc/nova/nova.conf

```diff
@@ -160,6 +160,11 @@ reserved_host_memory_mb = 4096
 {% endif %}
 disk_cachemodes = "network=writeback"
 
+{% if v7k.enabled %}
+iscsi_use_multipath = True
```

Comment:
should have flag to toggle multipath. Can we toggle between multi-path True/False and what happens to existing volume when we switch from True to False?